iothub_client/samples/pnp/readme.md

```diff
@@ -0,0 +1,23 @@
+# Samples to demonstrate Azure IoT Plug and Play
+
+The samples in this directory demonstrate how to implement a Azure IoT Plug and Play device.  Azure IoT Plug and Play is documented [here](aka.ms/iotpnp).  The samples assume basic familairity with PnP concepts, though not in depth knowledge of the PnP "convention".  The "convention" is a set of rules for serializing and deserialing data that uses  IoTHub primitives for transport.  The samples themselves demonstrate and have a sample helper library.
+
+## Directory structure
+
+The directory contains the following samples:
+
+* [pnp_simple_thermostat](./pnp_simple_thermostat) A simple thermostat that implements the model [dtmi:com:example:Thermostat;1](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json).  This sample is considered simple because it only implements one component, the thermostat itself.
+
+There is an additional helper directory, [common](./common), that is used by these samples.  Files in this directory:
+
```

Comment:
to remove from dir structure

Reply:
removing for now, will add this back once the component sample that uses helper goes in.